src/lib/config/defaultConfig.js

```diff
@@ -318,7 +318,18 @@ export const RECORDS_CONFIG = {
     },
   },
   DOCUMENT_REQUESTS: {
-    states: ['ACCEPTED', 'PENDING', 'DECLINED'],
+    states: [
+      {
+        text: 'ACCEPTED',
```

Comment:
minor: 
```suggestion
        value: 'ACCEPTED',
```
(and on the ones below)
because we tend to use `text` to as a display label, while these refer to actual values, and could be misleading in the context of the whole config file. Check for example `CIRCULATION.statuses`